internal/unsafecast/unsafecast.go

```diff
@@ -65,3 +65,12 @@ func BytesToSlice[T any](data []byte) []T {
 func BytesToString(data []byte) string {
 	return *(*string)(unsafe.Pointer(&data))
 }
+
+// StringToBytes applies the inverse conversion of BytesToString.
+func StringToBytes(data string) []byte {
```

Comment:
I'm so baffled that this is the recommended, best way to do this haha.

Reply:
I think `[]byte(s)` would be the recommended way, but it requires making a copy of the content since the byte slice is a mutable value.

The intent of this function is to avoid the copy.